src/Appwrite/Utopia/Database/Validator/Queries/Documents.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Appwrite\Utopia\Database\Validator\Queries;
+
+use Appwrite\Utopia\Database\Validator\IndexedQueries;
+use Appwrite\Utopia\Database\Validator\Query\Limit;
+use Appwrite\Utopia\Database\Validator\Query\Offset;
+use Appwrite\Utopia\Database\Validator\Query\Cursor;
+use Appwrite\Utopia\Database\Validator\Query\Filter;
+use Appwrite\Utopia\Database\Validator\Query\Order;
+use Utopia\Database\Database;
+use Utopia\Database\Document;
+
+class Documents extends IndexedQueries
+{
+    /**
+     * Expression constructor
+     *
+     * @param Document[] $attributes
+     * @param Document[] $indexes
+     */
+    public function __construct(array $attributes, array $indexes)
+    {
+        $validators = [
```

Comment:
Do `$id`, `$createdAt`, and `$updatedAt` need to be added to the attributes?

Reply:
I tried with and without, worked the same. So I removed it, probably doesn't need to be added. Will implement tests to ensure this is true.

Reply:
Nevermind. Required. Added.